_pages/leadership-integration-project.md

```diff
@@ -2,51 +2,53 @@
 ID: 1402
 post_title: Leadership Integration Project
 author: Colin Madland
-post_excerpt: ""
+post_excerpt: ''
 layout: page
-permalink: >
+permalink: |
   https://create.twu.ca/ldrs591-sp18/leadership-integration-project/
 published: true
-post_date: 2017-11-25 12:43:15
+post_date: 2017-11-25T12:43:15.000Z
 ---
-All of the work you have done in LDRS 591 supports the MA Leadership program capstone, the Learning Integration Project (LIP) (LDRS 697/698).  LDRS 591 is the main prerequisite for the culminating activity of the MA in Leadership program, LDRS 697/698.
+
```

Comment:
Hi Colin,  I am not sure what to do with this section.  It is really talking about a completely different course.  I don't think it should be included in 591.  My plan right now is to put this information on Moodle for LDRS 697/698.